wordpress-mobile/WordPress-Android
# Feature/toggle between preview modes

This PR is part of the feature branch. Additional functionality comes soon.

This PR adds the ability to toggle between desktop and native to devise view modes for responsive websites.

I stumbled upon a combination of WebView settings (pretty obvious in hindsight) that allowed me to avoid using JS injection to manipulate viewport.

Different website implements mobile/desktop versions in different ways, so we can only do our best to show them.
[![Image from Gyazo](https://i.gyazo.com/cf635611b5b63eeb167b4a1c76135756.gif)](https://gyazo.com/cf635611b5b63eeb167b4a1c76135756)
![Screenshot_1562888618](https://user-images.githubusercontent.com/728822/61092445-2686d480-a3fb-11e9-8183-6f5ee02bc53b.png)

To test:
- Navigate to site preview.
- Toggle desktop/default preview and make sure it works and persists on configuration change.
- You can toggle the preview mode before the page is loaded.

Update release notes:


I'm waiting for parent PR to get merged before this one is ready for code review. The design review can be performed right now.
